app/index.html

```diff
@@ -1,151 +1,502 @@
-            if (index !== -1) {
-              scope[model + 's'][index] = _.merge(scope[model + 's'][index], response.data);
-            }
-            //create
-            else {
-              scope[model + 's'].push(response.data);
-            }
-          }
-        })
-      );
-    };
-    io.sails.url = location.search.split('backend=')[1] || 'http://localhost:8080';
-    angular.module('wallboardApp', [
-      'wallboardApp.controllers', 'angularMoment'
-    ]);
-    angular.module('wallboardApp.controllers', []).
-      controller('wallboardController', ($scope) => {
-        io.socket.on('connect',  () => {
-          $scope.$apply(() => $scope.connected = true);
-          sailsBind('centre', 'centre_id', $scope, io);
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+    <title>Wall Board</title>
+    <script src="https://code.jquery.com/jquery-2.1.4.min.js" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"
+            integrity="sha256-Sk3nkD6mLTMOF0EOpNtsIry+s1CsaqQC1rVLTAy+0yc= sha512-K1qjQ+NcF2TYO/eI3M6v8EiNYZfA95pQumfvcVrTHtwQVDG+aHRqLi/ETn2uB+1JqwYqVG3LIvdm9lj6imS/pQ=="
+            crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css"
+          crossorigin="anonymous">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha256-MfvZlkHCEqatNoGiOXveE8FIwMzZg4W85qfrfIFBfYc= sha512-dTfge/zgoMYpP7QbHy4gWMEGsbsdZeCXz7irItjcC3sPUFtf0kuFbDz/ixG7ArTxmDjLXDmezHubeNikyKGVyQ=="
+          crossorigin="anonymous">
+
+
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700,400italic,700italic' rel='stylesheet'
+          type='text/css'>
+
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.7/angular.js" crossorigin="anonymous"></script>
+    <script src="https://cdn.rawgit.com/balderdashy/sails.io.js/master/dist/sails.io.js"
+            crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/3.10.1/lodash.min.js"
+            crossorigin="anonymous"></script>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.6/moment.min.js"
+            crossorigin="anonymous"></script>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-moment/1.0.0-beta.2/angular-moment.min.js"
+            crossorigin="anonymous"></script>
+    <script>
+        io.sails.url = location.search.split('backend=')[1] || 'https://api.ircbd.homeoffice.gov.uk:443';
+        io.sails.transports = ['polling'];
+        angular.module('wallboardApp', [
+            'wallboardApp.controllers', 'angularMoment'
+        ]);
+        angular.module('wallboardApp.controllers', []).
+        controller('wallboardController', function ($scope) {
```

Comment:
this has ended up with the old code, the stuff I wrote  recently with `sailsBind` is probably preferable to this.